runelite-client/src/main/java/net/runelite/client/plugins/cluescrolls/ClueScrollWorldMapPoint.java

```diff
@@ -26,15 +26,16 @@
 
 import java.awt.Graphics;
 import java.awt.image.BufferedImage;
+
 import net.runelite.api.Point;
 import net.runelite.api.coords.WorldPoint;
 import net.runelite.client.ui.overlay.worldmap.WorldMapPoint;
 
 class ClueScrollWorldMapPoint extends WorldMapPoint
 {
 	private static final BufferedImage CLUE_SCROLL_WORLD_IMAGE;
+	private static final BufferedImage CLUE_SCROLL_WORLD_IMAGE_EDGE;
 	private static final Point CLUE_SCROLL_WORLD_IMAGE_POINT;
-
```

Comment:
I believe there should be a blank line here. You should double check what other static blocks do